pom.xml

```diff
@@ -98,7 +98,7 @@
         <cglib_version>2.2</cglib_version>
         <mockito_version>2.23.4</mockito_version>
         <!-- Build args -->
-        <argline>-server -Xms256m -Xmx512m -XX:PermSize=64m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8
+        <argline>-server -Xms256m -XX:PermSize=64m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8
```

Comment:
Dubbo 2.7 runs on Java 8, in Java 8 we need Metaspace and MaxMetaspace.

Reply:
Hi, this is not resolved yet.

We need  `-XX:MetaspaceSize=64m -XX:MaxMetaspaceSize=128m` here.

Reply:
Thanks, if any further changes are required, please let me know